content/en/docs/contribute/style/style-guide.md

```diff
@@ -58,9 +55,9 @@ leads to an awkward construction.
 {{< table caption = "Do and Don't - API objects" >}}
 Do | Don't
 :--| :-----
-The Pod has two containers. | The pod has two containers.
-The Deployment is responsible for ... | The Deployment object is responsible for ...
-A PodList is a list of Pods. | A Pod List is a list of pods.
+The pod has two containers. | The Pod has two containers.
+The deployment is responsible for ... | The Deployment object is responsible for ...
```

Comment:
See? Here is where the confusion comes.
When you say "the deployment is responsible for ...", I have no clear idea what you are talking about. However, when you say "the Deployment is removed", I can clearly understand wha you mean.

Reply:
Agreed that "deployment" here may be both consistent to a convention and unhelpful to readers.

@jimangel @liggitt 👋 Any input here?

Reply:
I agree the "Do" example proposed here is confusing. I typically refer to API objects using the kind of the object (`Deployment`, `HorizontalPodAutoscaler`, etc), and I usually monospace it to make it clear it is a programmatic name. The previous guidance seems easier to apply consistently and less likely to lead to confusing constructions like this one.

Was there a problem with the previous guidance this is trying to correct? Maybe an additional example line in this table for `HorizontalPodAutoscaler` would be good to make sure the guidance we come up with works for the range of object names we'll need to apply it to.

Reply:
HorizontalPodAutoscaler is a good example. Some others, that in English are also sentence fragments and a good choice for the natural-language description of what the resource represents:
- CustomResourceDefinition
- PodSecurityPolicy
- PodDisruptionBudget

Reply:
In the DO example above, I would read that as referring to a general use of the term deployment. (There is a rule later talking about using monospaced font to denote a specific `Deployment` object, that clears things up, imo.) I would then think that the DON'T example, would have two problems, Capitalized, like an object, AND using the word "object".  In our docs I've seen more uses of the generic term deployment.

Reply:
Maybe it's just an example problem like you guys have mentioned. We should clarify those. In general, when I read this page and apply the new formatting, it makes sense and I don't see any confusing references in real world scenario:

https://kubernetes.io/docs/concepts/workloads/controllers/deployment/

Reply:
Ok, I've updated the examples to ensure every scenario is covered and used something different than deployment as an example. If you also think we need a deployment example that makes more sense than the original, I can add another row to the table.

Reply:
This works right up until someone copies your documentation and pastes it somewhere else. The only reliable means of describing an API object that can be understood unambiguously to be an object and can be read by a non-technical/novice reader (including a technical editor!) to be an object is to identify the object as an object in the text. 

This seems like overkill in the case of Pascal-cased objects such as those Tim mentions, but gets pretty hairy when there's no way to divine whether an object name is Pascal-cased or simply incorrectly capitalized. In other words, you need a standard that works even when someone who doesn't understand your capitalization rules gets their hands on the documentation.

Reply:
@wabernatScality

> This works right up until someone copies your documentation and pastes it somewhere else. 

This is a use case over which we have almost zero control. Optimizing for it seems like a lot of complication for a small impact.